### PR TITLE
Some tags in /api/packages/<pkg>

### PR DESCRIPTION
**Pull request by isoos**
- #3519
- not all tags are exposed, I'll update the issue with the related notes

### CODE REVIEW

File: pkg/client_data/lib/package_api.dart [hunk 1]
```diff
@@ -95,6 +95,9 @@ class PackageData {
   /// Package name.
   final String name;
 
+  /// Package-level tags
```
Comment (jonasfj) [inline pkg/client_data/lib/package_api.dart:98]: How about: `/// Tags applicable to all versions of this package`

File: pkg/client_data/lib/package_api.dart [hunk 2]
```diff
@@ -121,6 +125,9 @@ class PackageData {
 class VersionInfo {
   final String version;
 
+  /// Version-level tags.
```
Comment (jonasfj) [inline pkg/client_data/lib/package_api.dart:128]: `/// Tags applicable to a specific version of a package`

File: app/lib/package/backend.dart [hunk 3]
```diff
@@ -324,6 +324,7 @@ class PackageBackend {
   /// Used in `pub` client for finding which versions exist.
   Future<api.PackageData> listVersions(Uri baseUri, String package) async {
     return await cache.packageData(package).get(() async {
```
Comment (jonasfj) [inline app/lib/package/backend.dart:326]: We're changing the format of the data in the cache, let's bump the `runtimeVersion` for good measure.

**Comment by sigurdm** [comment]
Should we document these in https://github.com/dart-lang/pub/blob/master/doc/repository-spec-v2.md ?

File: app/lib/package/backend.dart (hunk 3 above)
Comment (isoos) [reply app/lib/package/backend.dart:326]: That's already happening due to the dartdoc version upgrade.

**Pull request merged**